.ci/jenkins/dockerfiles/ubuntu_16_04.dockerfile

```diff
@@ -45,22 +60,4 @@ RUN apt-get -y --no-install-recommends install \
     rm -rf /var/lib/apt/lists/*
 
 # Install tox
-RUN pip3 install tox==3.9.0
-
-# Build nGraph master
-ARG NGRAPH_CACHE_DIR=/cache
-
-WORKDIR /root
-RUN git clone https://github.com/NervanaSystems/ngraph.git
```

Comment:
Why are we removing this section?

Reply:
Incremental builds turned out to cause problems - failing to build nGraph during CI.